.github/workflows/create-release.yml

```diff
@@ -42,21 +42,40 @@ jobs:
     runs-on: ubuntu-latest
     needs: smoke
     steps:
+    - name: Checkout With History
+      uses: actions/checkout/@v2
+      with:
+        fetch-depth: 0 # gets full history
+    - name: Compare With Previous Release
+      id: compare_previous_release
+      run: |
+        output_string="::set-output name=builder_changes"
+        if [ -z "$(git diff $(git describe --tags --abbrev=0) -- builder.toml)" ]
+        then
+          echo "${output_string}::false"
+        else
+          echo "${output_string}::true"
+        fi
     - name: Checkout
+      if: ${{ steps.compare_previous_release.outputs.builder_changes == 'true' }}
       uses: actions/checkout@v2
-    - run: git fetch --depth=1 origin +refs/tags/*:refs/tags/* || true
+    - if: ${{ steps.compare_previous_release.outputs.builder_changes == 'true' }}
+      run: git fetch --depth=1 origin +refs/tags/*:refs/tags/* || true
     - name: Reset Draft Release
       id: reset
+      if: ${{ steps.compare_previous_release.outputs.builder_changes == 'true' }}
       uses: paketo-buildpacks/github-config/actions/release/reset-draft@main
       with:
         repo: ${{ github.repository }}
         token: ${{ github.token }}
     - name: Tag
       id: tag
+      if: ${{ steps.compare_previous_release.outputs.builder_changes == 'true' }}
       uses: paketo-buildpacks/github-config/actions/tag@main
       with:
         current_version: ${{ steps.reset.outputs.current_version }}
     - name: Create Release
```

Comment:
There is a canonical default action for this https://github.com/actions/create-release, what is the motivation behind using paketo actions here?

Reply:
See my [comment on resetting the draft release](https://github.com/paketo-buildpacks/full-builder/pull/10#discussion_r504635172).

Reply:
If there is no draft release, it looks like the reset-draft action won't output a current version. I am assuming the tag action handles an empty current version correctly?

Personally, I think it would be better to take out the reset draft action if we don't intend on using. It it not obvious to me, as someone who isn't intimately familiar with the internal of the custom actions, how exactly this workflow functions (without looking at the code). If we want to manually control semver bumping semantics I think we should reintroduce that later.

Reply:
The tag action does handle an empty version correctly. We can use it with the canonical Create Release action to cut releases that bump the patch version from the previous release.